packages/server/src/automations/actions.ts

```diff
@@ -15,6 +15,7 @@ import * as delay from "./steps/delay"
 import * as queryRow from "./steps/queryRows"
 import * as loop from "./steps/loop"
 import * as collect from "./steps/collect"
+import * as trigger from "./steps/trigger"
```

Comment:
As I said with my comment on the steps file, this is an example where its a little confusing, the first time I saw this I was very confused why the triggers were being pulled into the action list, but I get whats happening now!